open_tms_backend/open_tms_create_organisation/index.js

```diff
@@ -1,19 +1,73 @@
 import { shared } from '@appblocks/node-sdk'
+import { v4 as uuidv4 } from 'uuid'
 
 const handler = async (event) => {
   const { req, res } = event
-  const { sendResponse } = await shared.getShared()
+  const { sendResponse, validateRequestMethod, isEmpty, checkHealth, validateRequiredParams, prisma } =
+    await shared.getShared()
+  const requiredParams = ['name', 'display_name']
 
-  // health check
-  if (req.params.health === 'health') {
-    return sendResponse(res, 200, { message: 'Health check success' })
-  }
+  const tenant_id = '71df8bec-075a-48a0-9ea7-6d5f85de729c'
```

Comment:
instead of hardcoding tenant_id you can get the tenant_id from the tenants table using name 'Appblocks' as it is the default tenant